src/commands/game_commands/profile.ts

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable @typescript-eslint/dot-notation */
+import Eris from "eris";
+import dbContext from "../../database_context";
+import { getDebugLogHeader, getMessageContext, getUserTag, sendEmbed, sendInfoMessage } from "../../helpers/discord_utils";
+import BaseCommand, { CommandArgs } from "../base_command";
+import _logger from "../../logger";
+import { bold, friendlyFormattedDate } from "../../helpers/utils";
+
+const logger = _logger("profile");
+
+export default class ProfileCommand implements BaseCommand {
+    help = {
+        name: "profile",
+        description: "Shows your game stats.",
+        usage: "!profile",
```

Comment:
Should we include here that a username can be mentioned, or will it make it more confusing than it needs to be for the user?